src/pages/pokemon/PokemonList.tsx

```
@@ -0,0 +1,62 @@
+          <div className="detail-container">
+            <p className="detail-close" onClick={closeDetail}>
+              X
+            </p>
+            <div className="detail-info">
+              <img src={image} alt="pokemon" className="detail-img" />
+              <p className="detail-name"> {name}</p>
+            </div>
+            <div className="detail-skill">
+              <p className="detail-ability"> Ablities: </p>
+              {abilities?.map((ab: any) => {
+                return <div className=""> {ab.ability.name}</div>;
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)